src/components/RCLink/RCLink.tsx

```diff
@@ -46,6 +48,10 @@ const RCLink = ({
           },
         }
       }
+      // TODO: These were added in accessibility QA. Investigate adding these to the DS Link component.
+      role="link"
+      aria-disabled={disabled}
+      tabIndex={disabled ? -1 : null}
```

Comment:
I think the alternative to -1 should be 0, which would include the link in sequential tabbing order. [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex) says tabbing behavior with an absent tabIndex (I imagine null would be absent) "is determined by the user agent." I think it would be better to definitely have the link included in sequential tabbing.

Reply:
@dgcohen not sure if you saw this comment, but once this is addressed I will approve.

Reply:
Not sure if you saw but i added the 0 index back in